bin/p2-wipe-reality/main.go

```diff
@@ -19,6 +21,16 @@ a known state.
 `
 )
 
+func handleError(err error, message string) {
```

Comment:
I can remove it, it was meant for debugging why this utility wasn't working. Dealing with the unsafe errors in Consul isn't something I care to solve in this PR.